lib/vsc/administration/vo.py

```diff
@@ -166,13 +174,13 @@ def members(self):
     def _get_path(self, storage, mount_point="gpfs"):
         """Get the path for the (if any) user directory on the given storage."""
 
-        (path, _) = self.storage.path_templates[storage]['vo'](self.vo.vsc_id)
+        (path, _) = self.storage.path_templates[self.host_institute][storage]['vo'](self.vo.vsc_id)
```

Comment:
I am thinking we should change the structure of this thing and get the `institute` one level higher up, so we can have `self.storage = ...[host_institute]` and take it from there? Or make it a real class and use properties instead of a dict with keys?

Reply:
Need not happen here or now.

Reply:
no, you're right. It's not needed.

Reply:
ok, I will make another PR for this (and the GPFS 3.5 stuff) once this is merged.